sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -406,6 +406,31 @@ ze_result_t ZeCall::doCall(ze_result_t ZeResult, const char *CallStr,
     return mapError(Result);
 #define ZE_CALL_NOCHECK(Call) ZeCall().doCall(Call, #Call, false)
 
+// This helper function creates a pi_event and associate a pi_queue.
+// We need to increment the reference counter here to avoid pi_queue
```

Comment:
Please move this comment to where retain is made.

Reply:
Done. Thanks.